nav2_costmap_2d/plugins/costmap_filters/costmap_filter.cpp

```diff
@@ -189,29 +189,6 @@ bool CostmapFilter::transformPose(
   return true;
 }
 
-bool CostmapFilter::worldToMask(
```

Comment:
I think we discussed moving this back since its only used in the filters anyway?

Reply:
It is used in VO-server in different places, like [here](https://github.com/AlexeyMerzlyakov/navigation2/blob/aed5df9e2671101dd87d7ff1b714984f4d885ee7/nav2_map_server/src/vo_server/vector_object_server.cpp#L244) or [here](https://github.com/AlexeyMerzlyakov/navigation2/blob/aed5df9e2671101dd87d7ff1b714984f4d885ee7/nav2_map_server/src/vo_server/vector_object_shapes.cpp#L218C10-L218C10), so if we want to have this routine in costmaps, we also need to duplicate it for map server. So, I'd like to switch in favor of using it in `nav2_util` instead (in order to avoid boilerplate works in the future by keeping synched duplicating parts of code in different places).